src/runtime/providers/math.ts

```diff
@@ -63,6 +68,11 @@ function degreesToRadians(degrees:number){
 }
 
 class Add extends TotalFunctionConstraint {
+  resolveProposal(proposal, prefix) {
```

Comment:
At the moment the add function in the Math file is also providing the ability to concatenate strings to the language. If the decision is that Eve should always use string interpolation then this can be removed.

My opinion is that concatenating strings with a + symbol is such second nature to programmers that it would provide a real WTF moment to filter a block anytime "" + "" is used.